tests/test_utils/tests.py

```diff
@@ -418,25 +419,38 @@ def test_failure(self):
 
     def test_with_client(self):
         with CaptureQueriesContext(connection) as captured_queries:
-            self.client.get("/test_utils/get_person/%s/" % self.person_pk)
+            self.client.get(self.url)
         self.assertEqual(len(captured_queries), 1)
         self.assertIn(self.person_pk, captured_queries[0]["sql"])
 
         with CaptureQueriesContext(connection) as captured_queries:
-            self.client.get("/test_utils/get_person/%s/" % self.person_pk)
+            self.client.get(self.url)
         self.assertEqual(len(captured_queries), 1)
         self.assertIn(self.person_pk, captured_queries[0]["sql"])
 
         with CaptureQueriesContext(connection) as captured_queries:
-            self.client.get("/test_utils/get_person/%s/" % self.person_pk)
-            self.client.get("/test_utils/get_person/%s/" % self.person_pk)
+            self.client.get(self.url)
+            self.client.get(self.url)
         self.assertEqual(len(captured_queries), 2)
         self.assertIn(self.person_pk, captured_queries[0]["sql"])
         self.assertIn(self.person_pk, captured_queries[1]["sql"])
 
+    def test_with_client_nested(self):
```

Comment:
For me, when I revert the code changes this test and the other test (`AssertNumQueriesContextManagerTests.test_with_client_nested`) still pass :thinking:

Reply:
Ah really strange, now I get failures. Sorry about that :+1:

Reply:
No worries!